dataland-frontend/src/components/pages/SearchCompaniesForFrameworkData.vue

```diff
@@ -12,35 +12,50 @@
           <FrameworkDataSearchBar
             v-model="currentSearchBarInput"
             ref="frameworkDataSearchBar"
-            :searchBarName="searchBarName"
-            :frameworksToFilterFor="currentFilteredFrameworks"
+            :filter="currentCombinedFilter"
+            :search-bar-name="searchBarName"
+            :enable-full-search="true"
             @companies-received="handleCompanyQuery"
-            @rendered="handleFrameworkDataSearchBarRender"
-            class="pl-4"
+            @search-confirmed="handleSearchConfirmed"
           />
-
+          <div class="flex">
+            <FrameworkDataSearchFilters
+              :show-heading="true"
+              v-model:selected-country-codes="currentFilteredCountryCodes"
+              v-model:selected-frameworks="currentFilteredFrameworks"
+              v-model:selected-sectors="currentFilteredSectors"
+            />
+            <span class="d-page-display">{{ currentlyVisiblePageText }}</span>
+          </div>
           <div
-            :class="[
-              pageScrolled && !searchBarToggled
-                ? ['col-12', 'align-items-center', 'grid', 'bg-white', 'd-search-toggle', 'fixed', 'd-shadow-bottom']
-                : 'pl-2',
-            ]"
+            class="col-12 align-items-center grid bg-white d-search-toggle fixed d-shadow-bottom"
+            v-show="!searchBarToggled && pageScrolled"
           >
-            <span v-if="!searchBarToggled && pageScrolled" class="mr-3 font-semibold">Search Data for Companies</span>
+            <span class="mr-3 font-semibold">Search Data for Companies</span>
             <PrimeButton
-              v-if="!searchBarToggled && pageScrolled"
               name="search_bar_collapse"
               icon="pi pi-search"
               class="p-button-rounded surface-ground border-none m-2"
               @click="toggleSearchBar"
             >
               <i class="pi pi-search" aria-hidden="true" style="z-index: 20; color: #958d7c" />
             </PrimeButton>
+            <FrameworkDataSearchFilters
```

Comment:
The dropdown location is not adjusting to the "scrollToTop" situation when you choose something out of the multiselect.

Reply:
I fixed this